api/src/beacons/beacon.schema.ts

```diff
@@ -16,6 +23,9 @@ export class Beacon {
   @Prop({ required: true })
   description: string;
 
+  @Prop({ required: true })
+  tag: Tags;
```

Comment:
Have you tested this? I think it should be the following.
`@Prop({ required: true, enum: Object.values(Tags) })
tag: string;`

Reply:
I did test and it's covered by the restrictions on create-beacon and find-beacons but it wouldn't hurt to add here too.